src/BlockTypes/ProductRating.php

```diff
@@ -109,10 +110,13 @@ protected function render( $attributes, $content, $block ) {
 		);
 
 		if ( $product ) {
+			$classes_and_styles = StyleAttributesUtils::get_text_align_class_and_style( $attributes );
+
 			return sprintf(
-				'<div class="wc-block-components-product-rating wc-block-grid__product-rating">
+				'<div class="wc-block-components-product-rating wc-block-grid__product-rating %s">
 					%s
 				</div>',
+				$classes_and_styles['class'] ?? '',
```

Comment:
Just to be extra-cautious, I think we should escape this:
```suggestion
				esc_attr( $classes_and_styles['class'] ?? '' ),
```

Reply:
We should open a separate Issue if this is a concern, as multiple blocks are not escaping it at the moment. So, we probably should address this for all of them.

When I added escaping, did a quick test, it broke the alignment functionality, so it probably needs some more investigating how to tackle it correctly. I wouldn't want it to be blocking this PR though.